packages/Popover/src/components/Trigger/Trigger.js

```diff
@@ -45,10 +45,14 @@ function Trigger(props) {
     }
   }
 
-  const { children, a11yText } = props;
+  const { children, a11yText, ...moreProps } = props;
 
   if (typeof children === "function") {
-    return <React.Fragment>{children(handleTriggerEvent, { "aria-describedby": content.ariaId })}</React.Fragment>;
+    return (
+      <React.Fragment>
+        {React.cloneElement(children(handleTriggerEvent, { "aria-describedby": content.ariaId }), { ...moreProps })}
```

Comment:
so if the child of the trigger component is a `node` it won't receive the `...moreProps`. Is that intentional?

Reply:
oh i see you add below